SCClassLibrary/Common/Control/Server.sc

```diff
@@ -175,7 +175,7 @@ ServerOptions {
 		if (outputStreamsEnabled.notNil, {
 			o = o ++ " -O " ++ outputStreamsEnabled ;
 		});
-		if ((thisProcess.platform.name!=\osx) or: {inDevice == outDevice})
+		if (inDevice == outDevice)
```

Comment:
Perhaps `if((thisProcess.platform.name == \linux) or: { inDevice == outDevice })` ? Because separate IO devices won't behave the same in Linux.

Reply:
Wouldn't that depend on which backend is used? While we officially support JACK only, it seems possible to build with PortAudio, in which case we might want to have different I/O devices.

It seems that conceptually the switch here should depend on the _backend_, not the platform, but we have no way of checking that so I'd not put it in. Anyway, if the user specifies a single device (`s.options.device`), which is the intended use on linux (JACK) AFAIU, this should behave the same.

Reply:
OK, I can accept this as "user's responsibility."